solutions/c/02-cz2/code/.codecrafters/run.sh

```
@@ -1,12 +1,11 @@
-# 
+#
 # This runs after .codecrafters/compile.sh
 #
 # Learn more: https://codecrafters.io/program-interface
 
-# Exit early if any commands fail
-set -e
+set -e # Exit on failure
 
-exec /tmp/shell-target "$@"
+exec $(dirname $0)/build/shell "$@"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)